DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -344,5 +345,17 @@ public IEnumerable<Course> GetCoursesAtCentreForAdminCategoryId(int centreId, in
                 new { centreId, categoryId }
             );
         }
+
+        public (int? centreId, int? categoryId) GetCentreIdAndCategoryIdForCourse(int customisationId)

```

Comment:
Would it make sense to just pass in customisationId, categoryId and centreId into this method and return a bool? I can't see any use case for getting just centreId and categoryId for a customisation 🤔

Reply:
Changed the method to return a boolean if the course exists or not :)